<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(105, 127, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="table">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>220</y>
      <width>791</width>
      <height>361</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>781</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>255</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>255</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Black</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>!Enter movie name!</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_name">
       <property name="minimumSize">
        <size>
         <width>255</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>255</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_name">
       <property name="minimumSize">
        <size>
         <width>255</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>255</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Black</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(157, 165, 213);</string>
       </property>
       <property name="text">
        <string>!Press!</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalWidget" native="true">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>120</y>
      <width>781</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Black</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Year</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_year">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Black</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Time</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_time">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Black</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Genres</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combo">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>50</height>
        </size>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Не выбрано&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(203, 194, 207);</string>
       </property>
       <item>
        <property name="text">
         <string>не выбрано</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_genre">
       <property name="minimumSize">
        <size>
         <width>183</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>183</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Black</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(157, 165, 213);</string>
       </property>
       <property name="text">
        <string>!Press!</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
